### PR TITLE
Add expose doc

### PR DESCRIPTION
**Pull request by 0xmichalis**

### CODE REVIEW

**Comment by adellape** [comment]
Depends on https://github.com/openshift/origin/pull/2045

**Comment by kargakis** [comment]
@adellape ping

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -96,6 +96,12 @@ a|`osc delete -f _<file_path>_`
 `osc delete all -l _<label>_`
 .^|Deletes the specified OpenShift object. An object configuration can also be passed in through STDIN. The `osc delete all -l _<label>_` operation deletes all objects matching the specified `_<label>_`.
 
+|`expose`
+|`osc expose _<object_type>_ _<object_name_or_id>_`
+|Looks up a service and exposes it as a route. There is also the ability to expose a deployment config, replication controller, service, or pod as a new service on a specified port. If no labels are specified, the new object will re-use the labels from the object it exposes.
+
+|===
```
Comment (adellape) [inline cli_reference/basic_cli_operations.adoc]: This line can be removed (breaks the table currently).

**Comment by adellape** [comment]
Thanks, @kargakis. One comment I just noticed, then I can merge.

**Comment by kargakis** [comment]
ready

**Pull request merged**